### PR TITLE
feat: Add license info to radar entries

### PR DESCRIPTION
**Pull request by sasjo**
This PR introduces support for license information in radar entries. The license information will be displayed on the entry information page and also included as tags for filtering.

### CODE REVIEW

File: src/js/components/entry/License.jsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+import Icon from '../Icon';
+
+const OS_LICENSES = {
+  'Apache-1.1': 'Apache Software License Version 1.1',
+  'Apache-2.0': 'Apache Software License Version 2.0',
+  'BSD-2': 'Simplified BSD License (BSD 2-Clause License)',
+  'BSD-3': 'New BSD License (BSD 3-Clause License)',
+  'BSD-4': 'Original BSD License (BSD 4-Clause License)',
+  'BSD-style': 'BSD-style license, see project license for details',
+  CC0: 'Creative Commons Public Domain',
```
Comment (Tasseri) [inline src/js/components/entry/License.jsx]: Shouldn't these be in single quotations also?

Comment (Tasseri) [reply src/js/components/entry/License.jsx]: "CC0"

Comment (Tasseri) [reply src/js/components/entry/License.jsx]: "MIT" "Public"

Comment (sasjo) [reply src/js/components/entry/License.jsx]: Not unless we disable the eslint rule for that line. If the value doesn't need quotes, the rule says leave them out.

Comment (Tasseri) [reply src/js/components/entry/License.jsx]: Ok, I see! Thank you!

**Pull request merged**